build.gradle

```diff
@@ -8,6 +8,8 @@ buildscript {
     dependencies {
         classpath "com.android.tools.build:gradle:4.1.2"
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath "org.jetbrains.kotlin:kotlin-android-extensions:$kotlin_version"
+        testImplementation 'org.junit.jupiter:junit-jupiter:5.7.0'
```

Comment:
```suggestion
```
Wrong `build.gradle`. It should go in `app/`. That said there are no tests using JUnit 5 (there were at some point idk why) so you might as well nix that.